Formula/vips.rb

```diff
@@ -37,7 +38,7 @@ def install
       --disable-dependency-tracking
       --prefix=#{prefix}
       --with-magick
-      --with-magickpackage=GraphicsMagick
+      --with-magickpackage=ImageMagick
```

Comment:
The default seems to be set to MagickCore, do you know if there's a difference between the two ?

Reply:
Looks like there is no difference. vips will be linked with IM's MagickCore anyway. We can remove this line.

Reply:
Done